Samples/StyleTransfer/VideoEffect/StyleTransferEffectComponent/StyleTransferVideoEffect.cs

```diff
@@ -0,0 +1,236 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.Media.Effects;
+using Windows.Media.MediaProperties;
+using Windows.Foundation.Collections;
+using Windows.Graphics.DirectX.Direct3D11;
+using Windows.Graphics.Imaging;
+using System.Runtime.InteropServices;
+using Windows.Media;
+using Windows.AI.MachineLearning;
+using Windows.Storage;
+using Windows.Foundation;
+using System.Runtime.InteropServices.WindowsRuntime;
+
+namespace StyleTransferEffectComponent
+{
+    public sealed class StyleTransferVideoEffect : IBasicVideoEffect
+    {
+
+        private LearningModelSession _session;
+        private LearningModelBinding _binding;

```

Comment:
at some point it would be good to have these created based on a path inside the effect so the caller doesn thave to deal with ml